doc/modules/spamassassin.md

```diff
@@ -41,6 +42,9 @@ to be treated as normal rule:
 
 With this setting in `spamassassin` section, all rules whose scores are higher than
 `0.1` are treated not as atoms but as the complete rules and evaluated accordingly.
+(Note that the spamassassin module uses a score of 0 for rules with no score specified
+in the file; this behavior is different from SpamAssassin, which has a default score
+of 1 for some types of rules.)
```

Comment:
There is `unknown_weight` parameter in Rspamd that defines it. Indeed, it is 1.0 by default.

Reply:
Is `unknown_weight` taken into account when evaluating the `alpha` parameter in the spamassassin module? When I used a file that had a lot of rules with no `score` lines (with alpha set to 0.1), the rules didn't seem to take effect till I added scores.

Reply:
Alpha is for SA rules, it doesn't take into account `unknown_weight`.

Reply:
So `alpha` doesn't work on rules with no score line in the file. I've updated my note to try to explain that.

Reply:
Well, it is the minimum threshold for score, which is `0` in case of missing score. But then, if this symbol is added to Rspamd, it's score is defined by `unknown_weight`. I know, that it's confusing and I'm sorry for that...

Reply:
The confusing part is that in SpamAssassin, the default score isn't zero unless the rule name starts with a double underscore. So I would have expected SA rules with no `score` line to be treated the same as rules with a `score` line that specified 1 point.

https://cwiki.apache.org/confluence/display/SPAMASSASSIN/WritingRules

### Notes about rule scores
A few short words about the behavior of the "score" command.
- rules with a score set to 0 are not evaluated at all
- rules with no score statement will be scored at 1.0, unless 3 or 4 is true
- rules starting with a double underscore are evaluated with no score, and are intended for use in meta rules where you don't want the sub-rules to have a score.
- although intended for the sa development effort, any rule starting with T_ will be treated as a "test" rule and will be run with a score of 0.01 (nearly 0). This can be handy when testing rules so you don't have to create score lines for them if you think you're not going to keep them.